enlighten/MockManager.py

```diff
@@ -0,0 +1,157 @@
+import os
+import json
+import logging
+from typing import Optional
+
+from PySide2.QtWidgets import QFileDialog
+
+from .Spectrometer import Spectrometer
+from wasatch.WasatchDeviceWrapper import WasatchDeviceWrapper
+from wasatch.MockUSBDevice import MockUSBDevice
+from wasatch.DeviceID import DeviceID
+
+log = logging.getLogger(__name__)
+
+class MockManager:
+
+    # mock_defaults will be available for generated spectra option
+    mock_defaults = {"WP-785-ER": ("WP_785_ER", "WP_785_ER.json"),
+                     "NIR1": ("WP_00998_NIR1", "WP_00998.json")}
```

Comment:
Eventually I'd like this to be dynamically generated from the set of installed files, but okay for now.